### DISCUSSION TITLE
MantiCore can not be found in Linux after pip installation

### DISCUSSION THREAD

**Discussion by ferdasonmez** [trailofbits/manticore#2494] [Q&A] [answered]
Hi,
I installed Manticore using Pip. Unfortunetely, when I try to start the CLI I get this command not found error. Any suggestions?

Kind Regards,

**Comment by ferdasonmez** [comment]
Hi Again,
I installed the manticore to the linux path. Now I get permission error. I gave chmod 777 to manticore.py under ethereum. Still can not start anything. I would appreciate some help. Many thanks

**Comment by disconnect3d** [comment, marked as answer]
Hey, please use GitHub issues instead of `discussions` for issues.

If you did just `pip install manticore` without taking care of e.g. a Python virtual environment, the install likely ended with the following warning:
![image](https://user-images.githubusercontent.com/10009354/135375492-a49f13a1-5624-4000-b8e1-54381ef5b42d.png)

Which, as it says, means the script was installed to (in my case) `/home/dc/.local/bin` and this path needs to be added to the [Linux `$PATH` environment variable so your shell can find it when you type in `manticore`](https://www.baeldung.com/linux/path-variable).


We can see all of this below - in my case, after `pip install manticore` I am getting:
```
$ manticore
manticore: command not found
```

Which  can be fixed by adding the path the `manticore` script was installed to to the `PATH` envvar:
```bash
export PATH=$PATH:/home/dc/.local/bin
```

Afterwards the `manticore` command works for me:
```
$ manticore --version
Manticore 0.3.6
```

Note that it is good to add this `export PATH=...` line into your `~/.bashrc` file so it gets executed on each shell initialization.
